scripts/tf.sh

```diff
@@ -6,6 +6,11 @@
 
 ################### VARIABLES ############################################
 
+# The mlflow login information pulled from Google secrets to pass to the docker image for tracking
+export LOGNAME=$(gcloud config get-value account)
```

Comment:
if we parse this to end at the @ we could make the email address into a default Experiment Name